simplq/src/components/common/QrScanner/QrScanner.jsx

```diff
@@ -11,10 +11,10 @@ export default () => {
 
   const handleScan = (data) => {
     if (data != null) {
-      const res = this.getRoute(window.location.origin, data.text);
+      const res = getRoute(window.location.origin, data.text);
 
       if (res.verdict) {
-        this.props.history.push(res.targetRoute);
+        props.history.push(res.targetRoute);
```

Comment:
Can you change line 7 to `export default ({history}) => {`?

Reply:
Hi, thanks for this hint. I'm not a React dev :D 
This seems to work :D